### DISCUSSION TITLE
Page not visible in the left navigation when navigating to the document for menus with many elements

### DISCUSSION THREAD

**Discussion by maxkuzkin** [GitbookIO/community#309] [Feature Requests]
Our gitbook based documentation has navigation with lots of top level elements. When users are opening a link to the document that is relatively below in the navigation, they end up with a situation similar to the below —

<img width="1320" alt="image" src="https://github.com/GitbookIO/community/assets/3930620/ae0012ab-c2a7-4515-8e40-0d10f385f017">

I.e. the article is not actually visible in the left menu, users have to scroll down to "find" where it is. 

Would be very nice if left meny automatically "scrolled" to make current document visible, like Confluence and many other portals with the similar structure do as illustrated below —

<img width="1401" alt="image" src="https://github.com/GitbookIO/community/assets/3930620/792f538d-191e-46df-9f79-c4720c74f387">

Might sound like a minor thing, but typical usage pattern actually requires users to navigate to "nearby" documents, thus being able to see the context of where this document is helps a lot.

**Comment by aikoii** [comment]
Hey @maxkuzkin ☀️! We've addressed this issue—now, when you visit a page, the left menu will automatically scroll to the corresponding page.

**Reply by maxkuzkin** [reply]
Hi @aikoii - we've tested today and still see the same issue... Is there something that we need to do to apply this change to our documentation or it's supposed to get there automatically? You can see in our https://docs.platform.softwareone.com that the "Powered by" badge still takes precious space on screen while there's lots of scroll —
<img width="450" alt="image" src="https://github.com/user-attachments/assets/176ee4d9-e94d-48e7-9828-95d0deeb0a4d">
